build-appdirs.sh

```diff
@@ -31,13 +31,13 @@ cp resources/usr/share/metainfo/appimagetool.appdata.xml appimagetool.AppDir/usr
 rm -rf appimaged.AppDir/ || true
 mkdir -p appimaged.AppDir/usr/bin
 mkdir -p appimaged.AppDir/usr/lib
-cp -f build/appimaged appimaged.AppDir/usr/bin
-cp -f build/validate appimaged.AppDir/usr/bin
+cp -f build/out/usr/bin/appimaged appimaged.AppDir/usr/bin
+cp -f build/out/usr/bin/validate appimaged.AppDir/usr/bin
 mkdir -p appimaged.AppDir/usr/share/metainfo
 cp resources/usr/share/metainfo/appimaged.appdata.xml appimaged.AppDir/usr/share/metainfo/
 
 cp resources/AppRun appimaged.AppDir/
-find /usr -name "libarchive.so.*.*" -exec cp {} appimaged.AppDir/usr/lib/ \; > /dev/null 2>&1
+find /usr -name "libarchive.so.*.*" -exec cp "{}" appimaged.AppDir/usr/lib/ \; > /dev/null 2>&1
```

Comment:
Didn't know this was needed. Learning every day ;-)

Reply:
It isn't needed, but cleaner to do so, and escaping this also makes it a bit more "compatible" with non-bash shells.